tutorials/monitoring-demo/next-steps.include.md

```diff
@@ -1,4 +1,4 @@
 
-If you have your own NServiceBus solution that you would like to add monitoring to, go to [Adding monitoring to an existing NServiceBus solution](/tutorials/monitoring-setup/).
+## Ready to start monitoring your own system?
```

Comment:
DocsEngine does not allow questions in headers.

Reply:
Can we make it allowed?

Reply:
Not super-quickly. For now I'd replace it with an imperative. "Start monitoring your own system" Would that work at least short-term?